WcaOnRails/app/views/registrations/psych_sheet_event.html.erb

```diff
@@ -25,24 +25,38 @@
     </thead>
 
     <tbody>
-      <% @registrations.each do |registration| %>
+      <% current_pos = 0 %>
```

Comment:
Complex Ruby code should not be in the view.

Not sure what it does exactly, but I'm sure it can be done - and tested - better in the controller.